apple/Tests/TestCases/RET/TransactionManifestTests.swift

```diff
@@ -38,4 +38,106 @@ final class TransactionManifestTests: Test<TransactionManifest> {
         
         XCTAssertNoDifference(summary.addressesOfAccountsRequiringAuth, ["account_tdx_2_1288efhmjt8kzce77par4ex997x2zgnlv5qqv9ltpxqg7ur0xpqm6gk"])
     }
+	
+	func test_from_instructions_string_with_max_sbor_depth_is_ok() throws {
+		let instructionsString = """
+CALL_METHOD
```

Comment:
Shouldn't those tests be saved in an rtm file too?

Reply:
sure, can do it after merge, just wanted to get approval from you and Omar first. Also should generalize file location of RTM files, now there were tied to `execution_summary` (in that folder), but should probably be some layer up, maybe... top level? `$PROJECT_ROOT/examples/transactions` or is it strange to put (new folder) `transactions` next to the iOS example app? speaking of which, is it possible to put the Android example app in that folder? 

if not `$PROJECT_ROOT/examples/transactions`, maybe ``$PROJECT_ROOT/resources/transactions` ? `$PROJECT_ROOT/fixtures/transaction` and also put JSON test vectors in `fixtures/testvectors`??

Reply:
Unfortunatelly the android example app cannot go there. It has to be under the jvm directory with its own build.gradle file.

Regarding the manifests and dat files, the `testvectors` is something I remember @0xOmarA was following with kotlin engine toolkit when we were developing on that repo.